apps/emqx_resource/src/emqx_resource_manager.erl

```diff
@@ -765,18 +813,36 @@ handle_event(enter, _OldState, ?state_stopped = State, Data0) ->
 %% The following events can be handled in any other state
 %%--------------------------
 handle_event(
-    {call, From}, #add_channel{channel_id = ChannelId, config = Config}, State, Data
+    {call, From}, #add_channel{channel_id = ChannelId, config = Config}, State, Data0
+) ->
+    {Actions, Data} = handle_not_connected_add_channel(From, ChannelId, Config, State, Data0),
```

Comment:
nit: inconsistent naming?
m.b. handle_add_channel_not_connected

Reply:
:checkered_flag: (in the newer PR)